beagle/src/main/kotlin/br/com/zup/beagle/android/imagedownloader/DefaultImageDownloader.kt

```diff
@@ -31,12 +31,12 @@ internal class DefaultImageDownloader : BeagleImageDownloader {
 
     private val imageDownloader: ImageDownloader = ImageDownloader()
 
-    override fun download(url: String, imageView: ImageView, rootView: RootView) {
+    override fun download(url: String, imageView: ImageView                                                                 , rootView: RootView) {
         imageView.post {
             rootView.getLifecycleOwner().lifecycleScope.launch(CoroutineDispatchers.IO) {
                 try {
                     val formattedUrl = url.formatUrl().takeIf { it.isNotEmpty() } ?: url
-                    val bitmap = imageDownloader.getRemoteImage(
+                    val bitmap =                                                                            imageDownloader.getRemoteImage(
```

Comment:
<table data-meta="generated_by_android">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ZupIT/beagle-android/blob/63e259024c86375ec2e99d4ad345b51067968334/beagle/src/main/kotlin/br/com/zup/beagle/android/imagedownloader/DefaultImageDownloader.kt#L39"></span>Line detected that is longer than the defined maximum line length in the code style.</td>
    </tr>
  </tbody>
</table>